### PR TITLE
match cache size to request

### PR DESCRIPTION
**Pull request by davejrt**
We should match the size of the cache to the default request for ephemeral storage.

### CODE REVIEW

**Review by daxmc99 — COMMENTED** [review]
I would personally prefer we raise the requests to this value. Decreasing it may have unintended consequences

**Comment by davejrt** [comment]
@daxmc99 I've increased the requests and limits. Do you off hand know if having the request/limit set to the same value would result in this always being evicted? I suspect we should allow some some burst right?

File: base/symbols/symbols.Deployment.yaml [hunk 1]
```diff
@@ -61,11 +61,11 @@ spec:
         resources:
           limits:
             cpu: "2"
-            ephemeral-storage: "8Gi"
+            ephemeral-storage: "12Gi"
             memory: 2G
           requests:
             cpu: 500m
-            ephemeral-storage: "4Gi"
+            ephemeral-storage: "10Gi"
```
Comment (daxmc99) [inline base/symbols/symbols.Deployment.yaml:68]:
```suggestion
            ephemeral-storage: "10G"
```

File: base/symbols/symbols.Deployment.yaml [hunk 2]
```diff
@@ -61,11 +61,11 @@ spec:
         resources:
           limits:
             cpu: "2"
-            ephemeral-storage: "8Gi"
+            ephemeral-storage: "12Gi"
```
Comment (daxmc99) [inline base/symbols/symbols.Deployment.yaml:64]:
```suggestion
            ephemeral-storage: "12G"
```

**Review by daxmc99 — CHANGES_REQUESTED** [review]
I think this makes the most sense. 
This will make 10GB a requirement to schedule the pod though. 
The pod will only be evicted if it reaches 12 GB 
We should document this in the changelog

**Pull request merged**